### PR TITLE
Hardik-Ghori

### PR DESCRIPTION
**Pull request by Hardik-Ghori**

### CODE REVIEW

File: Animations/Hardik-Ghori.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: Hardik Ghori
```
Comment (jayskhatri) [inline Animations/Hardik-Ghori.md]: Hey @Hardik-Ghori 👋, Just one change!
this md file should be in your username folder only!

**Review by jayskhatri — CHANGES_REQUESTED** [review]
Every other thing except md file is okay👍

File: Animations/Hardik-Ghori.md (hunk 1 above)
Comment (Hardik-Ghori) [reply Animations/Hardik-Ghori.md]: hello jay,
Is it good now?

**Review by jayskhatri — APPROVED** [review]
Good to go👍

Congratulations on making the submission ✨

**Pull request merged**